packages/stable/src/api/3d/revisions3DApi.ts

```diff
@@ -29,6 +30,14 @@ export class Revisions3DAPI extends BaseResourceAPI<Revision3D> {
     this.nodes3DApi = new Nodes3DAPI(resourcePath, httpClient, map);
   }
 
+  /**
+   * Specify what fields in json responses should be parsed as Dates
+   * @hidden
+   */
+  protected getDateProps(): DatePropFilter {
+    return [['items'], ['createdTime']];
```

Comment:
since we don't have Date parsing tests for all apis, have you checked all the apis and their props manually as well?
(I know you had a generator) but it might've missed something

Reply:
checked that, might have missed something, hard to find out